sdk/tests/test_upload_file.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+import lusid_drive
+import os
+from unittest import mock
+from lusid_drive.utilities import upload_file
+from lusid_drive.rest import ApiException
+
+
+
+class TestDriveFileUpload(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.local_file_path = os.path.join(os.path.dirname(__file__), "data", "test_file.txt")
+    
+    def test_upload_file_positive(self) -> None:
+        """
+        Test that a file can be created successfully
+
+        :return: None
+        """
+        api_client_mock = lusid_drive.FilesApi()
+        api_client_mock.create_file = mock.MagicMock()
+
+        with open(self.local_file_path, mode="r") as file:
+            body = file.read()
+
+        x_lusid_drive_filename = 'test_file.txt' # str | File name.
+        x_lusid_drive_path = '/' # str | Drive path.
+       
+        upload_file(api_client_mock,x_lusid_drive_filename, x_lusid_drive_path, body)
+        api_client_mock.create_file.assert_called_once_with(x_lusid_drive_filename='test_file.txt', x_lusid_drive_path='/', content_length=77018, body=body)
```

Comment:
Can you use the variable values in the checks - i.e. x_lusid_drive_filename=x_lusid_drive_filename